src/components/Select/styles.scss

```diff
@@ -23,6 +23,10 @@ $color-border-botton-yellow: #fcf000;
     justify-content: flex-end;
     align-items: flex-start;
     height: 100%;
+    width: 100%;
+    border: none;
+    outline: none;
```

Comment:
`outline: none;` is not good pattern, it will break navigation from keyboard.
but it can be other issue, can left for now